.github/workflows/update-monorepo-lockfiles.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Lockfiles for Dependabot Monorepo PRs
+
+on:
+  pull_request:
+    paths:
+      - 'superset-frontend/packages/**/package.json'
+      - 'superset-frontend/plugins/**/package.json'
+    # Trigger this workflow when Dependabot creates a pull request
+    types: [opened, synchronize, reopened]
```

Comment:
I'd say this should only trigger on `opened`, not the others. We don't want it to trigger itself, even though I think `git push` should be a no-op when the files does not change on the second run.